tracing/src/macros.rs

```diff
@@ -878,9 +878,63 @@ macro_rules! event {
 /// }
 /// ```
 ///
+/// # Alternatives
+///
+/// `enabled!` queries subscribers with [`Metadata`] where
+/// [`is_event`] and [`is_span`] both return `false`. Alternatively,
+/// use [`enabled_event`] or [`enabled_span`] to ensure one of these
+/// returns true.
+///
+///
+/// [`Metadata`]: crate::Metadata
+/// [`is_event`]: crate::Metadata::is_event
+/// [`is_span`]: crate::Metadata::is_span
+/// [`enabled_event`]: crate::enabled_event
+/// [`enabled_span`]: crate::enabled_span
+///
 #[macro_export]
 macro_rules! enabled {
-    (target: $target:expr, $lvl:expr, { $($fields:tt)* } )=> ({
+    ($($rest:tt)*)=> (
+        $crate::_enabled!(kind: { $crate::metadata::Kind::HINT }, $($rest)*)
+    )
+}
+
+/// The same as [`enabled`], but queries subscribers with `Metadata` where
```

Comment:
nit: should maybe be
```suggestion
/// The same as [`enabled!`], but queries subscribers with `Metadata` where
```